contracts/Wrapped1155Factory.sol

```diff
@@ -135,16 +145,32 @@ contract Wrapped1155Factory is ERC1155Receiver {
             
             // assign name
             hex"7f",
-            "Wrapped ERC-1155", uint128(32),
+            // Chars in hex or UTF-8 = 8 bits => 2 hex numbers => 
+            // string + uint128(30) for 16 chars needs to padding zeros 
+            // and the length of the string
+            // leading zeros = (64 - (tokenName.length * 2)) / 2
+            // Wrapped ERC-1155GnosisProtocol = 31 * 2 zeros + length * 2 in hex
+            // (62 - (31 * 2)) = 0 zeros + length * 2 in hex
+            // "Wrapped ERC-1155", uint128(32), 
+            // Wrapped ERC-1155 = 16 zeros + length * 2 in hex
+            // WrappedERC-1155 = 15 * 2 zeros + length * 2 in hex
+            // (62 - (15 * 2)) = 16
+            //"WrappedERC-1155", hex"000000000000000000000000000000001E", 
+            tokenName, tokenNameOffset, 
             hex"600655",
             
             // assign symbol
             hex"7f",
+            // Chars in hex or UTF-8 = 8 bits => 2 hex numbers => 
+            // string + uint128(30) for 16 chars needs to padding zeros 
```

Comment:
I find this comment kind of confusing as it doesn't relate to the code. I would change it to something like:
`// uint232 is used to pad the data of "WMT" and length to a total of 32 bytes to comply to the standard solidity bytes encoding`

Reply:
I think these encoded 31-byte strings could be computed fairly easily with a helper function so the intent is a little clearer.

Reply:
I will remove this comments and add documentation to explain the push32 bytecode and the bytes32 format with the zero padding and length.